common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -265,7 +267,11 @@ def __init__(self, *args, **kwargs):
         if not self.sources_list:
             if 'sources' in self.xml_attributes and isinstance(self.xml_attributes['sources'], basestring):
                 self.sources_list = [
-                    self.location.course_key.make_usage_key_from_deprecated_string(item)
+                    # TODO: it is not clear why we are replacing the run here (which actually is a no-op
```

Comment:
Added a TODO, as @nasthagiri suggested.

For more details, see https://github.com/edx/edx-platform/pull/15814#discussion_r133196390